open_discussions/views.py

```diff
@@ -30,6 +30,7 @@ def index(request, **kwargs):  # pylint: disable=unused-argument  # noqa: ARG001
             and (is_admin_user(request) or is_staff_list_editor(request)),
             "is_learning_path_editor": user.is_authenticated
             and (is_admin_user(request) or is_learning_path_editor(request)),
+            "is_article_editor": is_admin_user(request),
```

Comment:
I thought about using a separate group like we do for `api/v1/learningpaths/` permissions, but I decided to stay simple until we have a more concrete use case around this.